spec/models/solr_document_spec.rb

```diff
@@ -11,15 +11,6 @@
     end
   end
 
-  describe "to_param" do
```

Comment:
Should this really be removed completely or should it check to see that id was assigned